miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/MiniApp.kt

```diff
@@ -45,6 +46,7 @@ abstract class MiniApp internal constructor() {
     @Throws(MiniAppNotFoundException::class, MiniAppHasNoPublishedVersionException::class, MiniAppSdkException::class)
     abstract suspend fun create(
         appId: String,
+        queryParams: String,
```

Comment:
This `queryParams` can have default value such as empty string so there will be no impact for current function implementation.

Reply:
Yeah, I have updated this part.